modules/battery/__init__.py

```diff
@@ -38,21 +39,101 @@ class Battery(TextApp):
     def __init__(self):
         super().__init__()
         self.pin = tidal.BUTTON_A
+        self.update_interval_ms = 10000 # Sample every 100ms
+        self.average_discharge = 0
+        self.sample_count = 0
+        self.max_time_seconds = 3600 # Maximum expected battery life in seconds
```

Comment:
One hour? That is a wildly low estimate given we use lightsleep and switch the screen off. I don't have a production spec battery so I can't provide any actual data, but I'd guess something more like a day. Of course, depends how much the user is actually using it...